spec/habitat_spec.cr

```diff
@@ -189,7 +189,7 @@ describe Habitat do
     FakeServer.settings.this_can_be_nil.should eq "not nil"
 
     FakeServer.settings.nilable_with_default.should eq "default"
-    FakeServer.configure { |settings| settings.nilable_with_default = nil }
+    FakeServer.configure(&.nilable_with_default=(nil))
```

Comment:
Reverting the Ameba update should also allow you to revert this change.

Reply:
Sure! Reverted in https://github.com/luckyframework/habitat/pull/70/commits/3d3d5827cfd075c27379be6dc5813627c5cf2354